packages/vuetify/build/webpack.base.config.js

```diff
@@ -24,13 +21,17 @@ const cssLoaders = [
 const plugins = [
   new FriendlyErrorsWebpackPlugin({
     clearConsole: true
+  }),
+  new webpack.DefinePlugin({
+    __VUETIFY_VERSION__: JSON.stringify(vuetifyPackage.version),
+    __REQUIRED_VUE__: JSON.stringify(vuetifyPackage.peerDependencies.vue)
```

Comment:
This has to be done with babel, the es5 and lib builds aren't run through webpack

Reply:
@KaelWD, that babel plugin doesn't support babel 7 :(
Okay, I'll make another one